### PR TITLE
Intro to ML using R

### PR DESCRIPTION
**Pull request by fpsom**
A first draft of the tutorial for the intro to ML using R. Content is currently the same as the course delivered at SIB (https://github.com/fpsom/2020-07-machine-learning-sib).

It will be adapted and trimmed down to fit 2-3 hours (as opposed to the two full days it's currently designed for).

### CODE REVIEW

**Comment by hexylena** [comment]
Fantastic! Is this all of the episodes or just a subset?

**Comment by fpsom** [comment]
Currently this contains all content from all episodes, merged into a single tutorial.  Given the size, I'd be cutting stuff down as much as I can, to better fit the expected duration.

**Comment by hexylena** [comment]
Sounds good! We could also split them up into multiple tutorials? But I definitely understand if you just want to record something shorter for GCC :)

**Comment by fpsom** [comment]
Hm. Given that we do have the material, it may be possible to split this into multiple ones. However, as a first iteration, I'd argue to have a shortened (probably) version, in order to really check the time necessary to deliver. 

After GCC (i.e. a first delivery of the content in this form), we should definitely revisit this. :)

**Comment by hexylena** [comment]
Works for me, let's do that :)

**Comment by hexylena** [comment]
We should import the images at some point, but otherwise looks ok to start

**Comment by fpsom** [comment]
I've made another pass at the material, and added also the images (hopefully in the correct path).

I think what is missing now is formatting, and another set of eyes to review content itself.

**Comment by hexylena** [comment]
@fpsom it seems you created this PR from the main branch, as a result I can't push back changes. I've got the formatting changes mostly done. Could you either 1) give me permissions on your fork to push commits, or 2) make a branch off of main with your changes, and then PR that branch?

**Comment by fpsom** [comment]
Thanks @hexylena ! I've just added you to the fork :)

**Comment by hexylena** [comment]
Ok, formatting looks better. There are some TODOs I've left in the tutorial for whenever anyone has time

File: topics/statistics/tutorials/intro-to-ml-with-r/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,1262 @@
...
+A common practice in all supervised learning is the construction and use of the **train- and test- datasets**. This process takes all of the input randomly splits into the two datasets (training and test); the ratio of the split is usually up to the researcher, and can be anything: 80/20, 70/30, 60/40...
+
+## Supervised Learning I: classification
+
+There are various classifiers available:
+
+- **Decision Trees** – These are organized in the form of sets of questions and answers in the tree structure.
+- **Naive Bayes Classifiers** – A probabilistic machine learning model that is used for classification.
+- **K-NN Classifiers** – Based on the similarity measures like distance, it classifies new cases.
+- **Support Vector Machines** – It is a non-probabilistic binary linear classifier that builds a model to classify a case into one of the two categories.
```
Comment (anuprulez) [inline topics/statistics/tutorials/intro-to-ml-with-r/tutorial.md:797]: I think using the `kernel trick`, SVMs learn non-linear boundaries as well by projecting data points to higher-dimensional space. There are both linear and non-linear variants of SVMs. Thanks!

**Review by anuprulez — COMMENTED** [review]
@fpsom The tutorial looks great! I have added a few comments. Thanks!

**Comment by fpsom** [comment]
Thank you @anuprulez for the very positive and constructive feedback. I think I've addressed all your points in this new commit. :)

**Review by anuprulez — APPROVED** [review]

**Comment by anuprulez** [comment]
I guess some more work is remaining for the tutorial as the title suggests? Or WIP can be removed?

**Comment by fpsom** [comment]
Thanks @anuprulez ! I've just removed the WIP. I think the material is in a good enough state as they are. I do plan to make some updates (mostly in the solutions of the exercises), but I'm not sure that it'll be in the next few weeks.

**Comment by hexylena** [comment]
Yeah, there are a couple TODOs, but they can wait. The only one that should be fixed is probably:

>    TODO: (_There is a lot of information to unpack here, so we will discuss during the course what these 5 lines of code actually do_)

and those 5 lines aren't discussed afaict. (Is it even necessary?)

**Comment by fpsom** [comment]
Hm... I actually missed this ToDo...

Well, I don't really think that it would make sense to explain these here. If anything, a link to the `Visualization in R` tutorial could be included. Otherwise, I feel that this will boil down to a ggplot discussion.

But I'm open to suggestions - I can add a few sentences there, but (as I said), they might feel rather disjoined.

**Comment by hexylena** [comment]
Sounds good to me

**Comment by fpsom** [comment]
Perfect - done! :)

**Comment by fpsom** [comment]
Hm... Did I do something wrong now by pushing another commit? Referring to the "dismissed anuprulez’s stale review" notice above..

**Comment by hexylena** [comment]
No, that's normal. Every commit you push after someone has reviewed clears old reviews to ensure that no unreviewed content gets merged. Everything must have a review as the last step

File: topics/statistics/tutorials/intro-to-ml-with-r/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,1274 @@
...
+- Understand the ML taxonomy and the commonly used machine learning algorithms for analysing "omics" data
+- Understand differences between ML algorithms categories and to which kind of problem they can be applied
+- Understand different applications of ML in different -omics studies
+- Use some basic, widely used R packages for ML
+- Interpret and visualize the results obtained from ML analyses on omics datasets
+- Apply the ML techniques to analyse their own datasets
+time_estimation: 3H
+key_points:
+- To be added
+subtopic: r
```
Comment (hexylena) [inline topics/statistics/tutorials/intro-to-ml-with-r/tutorial.md:46]:
```suggestion
```

**Comment by fpsom** [comment]
Ah, clear now. Thanks @hexylena  ! :)

**Review by hexylena — APPROVED** [review]

**Pull request merged**